### DISCUSSION TITLE
Add DwCA metadata tool

### DISCUSSION THREAD

**Discussion by beach53** [specify/specify7#5529] [Feature Requests]
Edit by @grantfitzsimmons:
> We need to have a tool to create the EML metadata that the export process _currently_ requires. 
>
> https://ipt.gbif.org/manual/en/ipt/latest/gbif-metadata-profile

**Comment by grantfitzsimmons** [comment]
https://discourse.specifysoftware.org/t/dwca-data-exporting-in-specify-7/568

Still would be nice.
